src/main/resources/org/cbioportal/legacy/persistence/mybatis/GenePanelMapper.xml

```diff
@@ -186,4 +186,17 @@
             </choose>
         </where>
     </select>
+
```

Comment:
@fuzhaoyuan  it's not ANY panel, it's the panels which are currently at play in the queried studies, right?   a gene might be profiled by some gene panel that is totally unassociated with the current query